components/x-eventpromo/stories/index.js

```diff
@@ -0,0 +1,35 @@
+const { Eventpromo } = require('../');
+
+exports.component = Eventpromo;
+exports.package = require('../package.json');
+exports.stories = [
+	require('./eventpromo'),
+];
+
+exports.knobs = (data, { text }) => ({
+	dates() {
+		return text('Dates', data.dates);
+	},
+	eventpromoId() {
+		return text('Eventpromo Id', data.eventpromoId);
+	},
+	eventpromoLink() {
+		return text('Eventpromo link', data.eventpromoLink);
+	},
+	eventpromoTitle() {
+		return text('Eventpromo title', data.eventpromoTitle);
```

Comment:
can these props not be named `id`, `link` (or `url`) and `title` rather than requiring the `eventpromo` prefix?